src/core/parser.zsh

```diff
@@ -162,7 +162,7 @@ _fzf_complete_parse_option() {
         fi
     done
 
-    if [[ -z $result ]]; then
+    if [[ $#result = 0 ]]; then
```

Comment:
Why is this changed? In `_fzf_complete_kubectl_default_arguments` `-z` is used, so I wonder at those differences.

Reply:
This specific change is just for consistency with the rest of the similar implementations in this file. However, the change in `_fzf_complete_parse_option_arguments` is to fix a bug where it previously returned `1` when the given types were `'argument'` and the provided command was like `-n ""` whose result should have been `('')` that was considered empty.